fenix/app/src/main/java/org/mozilla/fenix/shopping/middleware/ReviewQualityCheckNavigationMiddleware.kt

```diff
@@ -4,21 +4,29 @@
 
 package org.mozilla.fenix.shopping.middleware
 
+import android.content.Context
+import androidx.annotation.VisibleForTesting
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.launch
+import mozilla.components.feature.tabs.TabsUseCases
 import mozilla.components.lib.state.Middleware
 import mozilla.components.lib.state.MiddlewareContext
+import org.mozilla.fenix.settings.SupportUtils
 import org.mozilla.fenix.shopping.store.ReviewQualityCheckAction
 import org.mozilla.fenix.shopping.store.ReviewQualityCheckState
 
+private const val PLACEHOLDER_URL = "www.fakespot.com"
```

Comment:
Think we can call the constant "POWERED_BY_URL", even if the url changes, we don't need to change the code then, only this link.